buildSrc/src/main/java/Testing.kt

```diff
@@ -0,0 +1,30 @@
+object Testing {
+    private const val junitVersion = "4.13.2"
+    const val junit4 = "junit:junit:$junitVersion"
+
+    private const val junitAndroidExtVersion = "1.1.3"
+    const val junitAndroidExt = "androidx.test.ext:junit:$junitAndroidExtVersion"
+
+    private const val coroutinesTestVersion = "1.5.1"
+    const val coroutines = "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutinesTestVersion"
+
+    private const val truthVersion = "1.1.3"
+    const val truth = "com.google.truth:truth:$truthVersion"
+
+    private const val mockkVersion = "1.10.0"
+    const val mockk = "io.mockk:mockk:$mockkVersion"
+    const val mockkAndroid = "io.mockk:mockk-android:$mockkVersion"
```

Comment:
Are we using mockK or mockito?

@Olamicode @osualaFC @kolanse 

I think we can stick to Mockito

Reply:
Since we're using Kotlin, mockK should be good. No need for Mockito.